app.go

```diff
@@ -71,6 +71,7 @@ func root() *cobra.Command {
 	rootCommand.Flags().StringVarP(&opts.GoTemplate, "template", "t", slice.DefaultTemplateName, "go template used to generate the file name when creating the resource files in the output directory")
 	rootCommand.Flags().BoolVar(&opts.DryRun, "dry-run", false, "if true, no files are created, but the potentially generated files will be printed as the command output")
 	rootCommand.Flags().BoolVar(&opts.DebugMode, "debug", false, "enable debug mode")
+	rootCommand.Flags().BoolVar(&opts.Quiet, "quiet", "q", false, "if true, no output is written to stdout/err")
```

Comment:
This should be:

```suggestion
	rootCommand.Flags().BoolVarP(&opts.Quiet, "quiet", "q", false, "if true, no output is written to stdout/err")
```

The `BoolVar` doesn't support shorthand.